backend/src/modules/tournament-tables/core/infrastructure/repository/inmemory/InMemoryTournamentTablesRepository.ts

```diff
@@ -0,0 +1,27 @@
+import { TournamentTablesRepository } from '../../../application/TournamentTablesRepository';
+import { TournamentTable } from '../../../domain/TournamentTable';
+
+export class InMemoryTournamentTablesRepository implements TournamentTablesRepository {
+  private readonly entities: { [tableId: string]: TournamentTable } = {};
+
+  async save(tournamentTable: TournamentTable): Promise<void> {
+    const tableId = `${tournamentTable.tournamentId}_${tournamentTable.tableNumber}`;
+    this.entities[tableId] = tournamentTable;
+  }
+
+  async saveAll(tournamentTables: TournamentTable[]): Promise<void> {
+    tournamentTables.forEach((table) => this.save(table));
```

Comment:
Obawiam się, że to nie będzie działać jak podepniesz prawdziwą bazę. 
Ale tak tylko Ci wspominam, żebyś wtedy nie waliła się z tym za długo, a teraz można mergować, bo problem się jeszcze nie objawił :) 

Odniosę się tylko, że to jest dokładnie to samo co mieliśmy w przypadku projektu ze Star Wars (żeby nie było, że te projekty są nie przydatne, a używa się te same zasady w prawdziwych aplikacjach) :)